man/docker.1.md

```diff
@@ -223,6 +223,70 @@ inside it)
   Block until a container stops, then print its exit code
   See **docker-wait(1)** for full documentation on the **wait** command.
 
+# CONFIGURATION FILES
```

Comment:
Another PR. @vdemeester am I volunteering or being volunteered. How about I make a PR and you remove it here?